src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateCollectionCommandResolver.java

```diff
@@ -497,6 +500,26 @@ private Integer validateModelAndDimension(
       EmbeddingProvidersConfig.EmbeddingProviderConfig providerConfig,
       Integer userVectorDimension) {
     // Find the model configuration by matching the model name
+    // 1. huggingfaceDedicated does not require model, but requires dimension
```

Comment:
For missing model: instead of adding hard-coded logic, would it be better to allow missing model if configuration specifies no models (which is the case for HuggingFace Dedicated)?
That way it is more configuration driven.

Similarly would be good to make "dimensions required" a property of configuration setting, but maybe that is more difficult do do.